NetKAN/FirespitterCore.netkan

```diff
@@ -14,11 +14,12 @@
     },
     "install"  : [
         {
-            "file"       : "Firespitter/Plugins",
+            "find"       : "Firespitter/Plugins",
             "install_to" : "GameData/Firespitter"
         },
         {
-            "file" : "Firespitter/Firespitter.version",
+            "find" : "Firespitter/Firespitter.version",
```

Comment:
shouldn't this be just Firespitter.version seeing as we're matching against files? Will there ever be a release with two .version files?